src/fps/proposal/IProposal.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity 0.8.15;
+
+import {AddressRegistry as Addresses} from "src/fps/AddressRegistry.sol";
+
+abstract contract IProposal {
```

Comment:
I wonder do we want to call these things 'Tasks' instead of 'Proposals'. I know this terminology is brought in from FPS so I understand it's origins.

Anyway, I don't think the change needs to be made in this PR. Just a callout for the future. Internally we don't really use the term 'proposal' for these things.

Reply:
+1 for tasks in a future PR

Reply:
Tracking this [here](https://github.com/ethereum-optimism/superchain-ops/issues/475) and resolving.